corehq/apps/users/models.py

```diff
@@ -2096,6 +2096,11 @@ def __repr__(self):
             self=self
         ))
 
+    @property
+    @memoized
```

Comment:
that's a good point, they seem to do the same exact thing. seems like we should just pick one and use it throughout